### ISSUE TITLE
strusPattern: %LEXER CASELESS and rule tokens

### ISSUE THREAD

**Issue by andreasbaumann** [patrickfrey/strusUtilities#64]
```
%LEXER CASELESS;

W = /aBc/;

R1 = any( W "aBc" );
R2 = any( W "abc" );
```

R1 matches, R2 not. Also 'ABC' in the text doesn't match do any R1, R2.
I suspect, the CASELESS has effect only on the Hyperscan option for case-insensitive checking.
IMHO it should also have an effect on the tokens..

**Comment by patrickfrey** [comment]
That's correct. Symbols should be matched case sensitively too in this case. This is a bug.
